spec/lib/avalon/bib_retriever_spec.rb

```diff
@@ -77,6 +77,28 @@
         expect(Nokogiri::XML(response)).to be_equivalent_to(mods)
       end
     end
+
+    describe 'multiple configured queries' do
+      let(:sru_url) { "http://zgate.example.edu:9000/db?version=1.1&operation=searchRetrieve&maximumRecords=1&recordSchema=marcxml&query=cql.serverChoice=%27%5EC%5E%257763100%27" }
```

Comment:
I think it would be clearer if you swap sru_url and sru_url2 since sru_url2 is what is queried first according to the order in the Avalon::Configuration.